src/styles/blocks/header.scss

```
@@ -0,0 +1,140 @@
+$header-height: 100vh;
+$header-item-height: 24px;
+$top-indent: 26px;
+
+.header {
+  position: relative;
+  height: $header-height;
+  background-image: url("./../images/header.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+ 
+  &__container {
+    display: flex;
+    height: $header-height;
+    padding: 0 24px;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: salmon;
+  }
+
+  &__info {
+    display: flex;
+    height: 8%;
+    padding-top: $top-indent;
+    justify-content: space-between;
+    align-items: center;
+  }
+     
+  &__menu-icon,
+  &__contacts-icon {
+    width: $header-item-height;
+    height: $header-item-height;
+    cursor: pointer;
+  }
+  
+  &__menu-icon {
+    background-image: url("./../images/icons/menu-standard.svg");
+    background-repeat: no-repeat;
+    transition: background-image 100ms linear;
+  }
+
+  &__menu-icon:hover {
+    background-image: url("./../images/icons/menu-open.svg");
+  }
+ 
+  &__contacts-icon {
+    height: $header-item-height;
+    position: relative;
+    background-image: url("./../images/icons/phone.svg");
+    background-repeat: no-repeat;
+  }
+
+  &__contacts-icon:hover::before {
+    content: "+ 1 234 555-55-55";
+    position: absolute;
+    right: 20px;
+    width: 140px;
+    font-weight: bold;
+    font-size: 12px;
+    line-height: $header-item-height;
+    letter-spacing: 2px;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  &__title {
+    position: relative;
+    max-width: 442px;
+    padding-bottom: 80px;
+ 
+    &--text {
+      margin: 24px 0;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 80px;
+      line-height: 80px;
+      letter-spacing: -1.5px;
+      text-transform: capitalize;
+      color: #fff;
+    }
+  }
+
+  &__title::before {
+    position: absolute;
+    top: 0;
+    content: "";
+    width: 40px;
+    height: 2px;
+    background: #fff;
+  }
+}
+
+.nav {
```

```suggestion
.nav {
```
`.nav` is a separate block so it should be in the separate file